runhouse/servers/http/http_server.py

```diff
@@ -68,6 +71,9 @@
 
 app = FastAPI(docs_url=None, redoc_url=None)
 
+# TODO: Better way to store this than a global here?
+running_futures: Dict[str, Future] = {}
```

Comment:
not sure if it's really any better, but could store it in the FastAPI app state, since this is only useful for the life of the server anyway right?